seqr/views/apis/data_manager_api.py

```diff
@@ -387,14 +387,75 @@ def load_rna_seq_sample_data(request, sample_guid):
         data_by_gene = json.loads(row.split('\t\t')[1])
 
     model_cls = RNA_DATA_TYPE_CONFIGS[data_type]['model_class']
-    models = model_cls.objects.bulk_create([model_cls(sample=sample, **data) for data in data_by_gene.values()])
-    logger.info(f'create {len(models)} {model_cls.__name__}', request.user, db_update={
-        'dbEntity': model_cls.__name__, 'numEntities': len(models), 'parentEntityIds': [sample_guid], 'updateType': 'bulk_create',
-    })
+    model_cls.bulk_create(request.user, [model_cls(sample=sample, **data) for data in data_by_gene.values()])
 
     return create_json_response({'success': True})
 
 
+def _log_append_info(user, info, message):
+    info.append(message)
+    logger.info(message, user)
+
+
+@data_manager_required
+def load_phenotype_prioritization_data(request):
+    request_json = json.loads(request.body)
+
+    file_path = request_json['file']
+
+    info = []
```

Comment:
You are misusing `info` in this function. The user is going to get all the messages at once, at the very end as a summary. So intermediate messages are just unhelpful clutter - the user wants to see summary statistics not "Attempting to load..", "Loaded ...". These are all potentially useful as log messages because they are logged in real time not at the end, but its probably a bit overkill especially since the bulk create/delete helpers are logging whats happening as well. The end result for info probably should look something like this:
```
- Loaded {tool} data in {num_projects} projects from {file_path}
- Project {project_a}: loaded {x} records, deleted {y} existing records
- Project {project_b}: loaded {n} records, deleted {m} existing records
- etc
```

Reply:
Okay